### ISSUE TITLE
Model doesn't appear to be using all available GPUs 

### ISSUE THREAD

**Issue by rdev** [replicate/dreambooth#9]
I ran training on this model with Cog on a 1xA100 instance and 8xA100 instance and got essentially the same performance. Is the container utilizing all available GPUs?

cc @chenxwh

**Comment by chenxwh** [comment]
I don't think it implements multi-gpu training? So essentially just on gpu is used
